src/MeetingsSDKAdapter.js

```diff
@@ -1362,10 +1358,6 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
         map((event) => this.attachMedia(ID, event)),
       );
 
-      const meetingWithMediaStoppedEvent$ = fromEvent(sdkMeeting, EVENT_MEDIA_STOPPED).pipe(
```

Comment:
being an SDK event, it might get triggered by the sdk, let's leave this handler here.

What we would need to do would be to see which stream has been stopped, and only set to null that one, not all media streams (audio, video, share). But we can deal with this in a different task.